src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_parser.py

```diff
@@ -262,44 +270,42 @@ def _get_measurement_document(
                         ]
                     ),
                     absorbance=TQuantityValueMilliAbsorbanceUnit(
-                        _get_float(data, row, "A260")
+                        _get_float(data, row, "a260")
                     ),
                 )
             )
 
-        a280_col = "A280"
-        if a280_col not in data.columns and "A280 10mm" in data.columns:
-            a280_col = "A280 10mm"
-
         if _get_str_or_none(data, row, a280_col):
             # capture concentration on the A280 measurement document if the experiment type is
-            # something other than DNA or RNA
+            # something other than DNA or RNA or if the experiment type is not specified
             mass_concentration = None
-            if na_type is not None and "NA" not in na_type and concentration_col:
+            if (na_type is not None and "NA" not in na_type and concentration_col) or (
+                na_type is None and concentration_col
+            ):
```

Comment:
Shorter, easier to read statement:
```suggestion
            if concentration_col and ("NA" not in (na_type or "")):
```